root/static/scripts/common.js

```diff
@@ -42,6 +42,7 @@ require('./common/components/WorkArtists');
 require('./common/MB/Control/SelectAll');
 require('./common/components/TagEditor');
 require('./common/components/sidebar/AcousticBrainz');
+require('./common/components/CritiqueBrainzReview');
```

Comment:
let's add some bundles for root/$entity/reviews.js and only load this in those files if possible -- I'm doing a bunch of work on removing common.js and edits.js so I'd prefer not to add anything new here

Reply:
So, just `manifest.js('reviews')` or whatnot?

Reply:
Yeah. If a single reviews.js file works for all entity types even better